test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -449,10 +465,20 @@ public void testSingleDocSqlResultSetHasOneRecord() throws SQLException {
             new GroupPrincipal("dgroup2"),
             new GroupPrincipal("dgroup1")))
         .build();
-    ResultSet rs = executeQuery("select * from data");
-    rs.next();
-    Acl acl = DatabaseAdaptor.buildAcl(rs, ",", DEFAULT_NAMESPACE,
-        ResultSetNext.STOP_NEXT);
+
+    Map<String, String> configEntries = new HashMap<String, String>();
+    configEntries.put("db.uniqueKey", "id:int");
+    configEntries.put("db.everyDocIdSql", "select id from data");
```

Comment:
Follow the boilerplate and put db.everyDocIdSql after the "Required for validation, ..." comment.